cves/CVE-2017-14056.yml

```diff
@@ -199,8 +217,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: No specifications are present in the code or commits.
```

Comment:
Even though none were explicitly listed, wouldn't the RL2 video have its own specification that should've been followed to implement it?